app/controllers/application_controller.rb

```diff
@@ -2,10 +2,16 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  helper_method :current_user
 
   def login_required
     if session[:user_id]
       @current_user = User.find(session[:user_id])
     end
   end
+
+  def current_user
```

Comment:
__[rubocop]__
* Use `attr_reader` to define trivial reader methods. [:ref](https://github.com/bbatsov/ruby-style-guide#attr_family)